src/QueryBuilder/QueryBuilder/QueryBuilderModal/RepeatableFields/RepeatableFields.js

```diff
@@ -114,6 +119,27 @@ export const RepeatableFields = ({ source, setSource, getParamsSource, columns }
     }));
   };
 
+  useEffect(() => {
+    if (calloutCalledRef.current) return;
+
+    const deletedFields = findMissingValues(fieldOptions, source);
+
+    if (deletedFields.length >= 1) {
+      calloutCalledRef.current = true;
+
+      callout({
+        type: 'warning',
+        message: (
+          <FormattedMessage
+            id="ui-plugin-query-builder.warning.deletedField"
+            values={{ value: deletedFields.join(',') }}
```

Comment:
please use `intl.formatList(...)` here, it will add commas (or other separators) and `and`, etc, as needed based on # of items and user locale

Reply:
@ncovercash , fixed! Thanks for that idea